web/main.ts

```diff
@@ -45,6 +48,24 @@ function saveBoundsSoon() {
   }, 1000);
 }
 
+function loadFrontend() {
+  numberRetries--;
```

Comment:
I think we should remove this retry logic. It doesn't provide any value in practice since it is a tight loop. It would be better if we could give the user the option to retry when ready, this lets them (me) go run `npm run start` in cases when I had forgot

Reply:
Yeah, makes sense - removed.